src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float Load_Factory = 0.75f;
+    private static final int MAGNIFICATION_FACTOR = 2;
+    private Node<K,V>[] table;
+    private int size;
+    private int capacity;
```

Comment:
let's remove this field
we don't actually need it
because we can call `table.length` in code where we need capacity